components/organisms/o-my-account-orders-history.vue

```diff
@@ -0,0 +1,180 @@
+<template>
+  <div class="o-my-account-orders-history">
+    <SfTabs :open-tab="1">
+      <SfTab :title="$t('My orders')">
+        <p class="message">
+          {{ $t('Check the details and status of your orders in the online store. You can also cancel your order or request a return.') }}
+        </p>
+        <div v-if="orders.length === 0" class="no-orders">
+          <p class="no-orders__title">
+            {{ $t('You currently have no orders') }}
+          </p>
+          <p class="no-orders__content">
+            {{ $t('Best get shopping pronto...') }}
+          </p>
+          <SfButton class="no-orders__button">
+            {{ $t('Start shopping') }}
+          </SfButton>
+        </div>
+        <SfTable v-else class="orders">
+          <SfTableHeading>
+            <SfTableHeader
+              v-for="tableHeader in tableHeaders"
+              :key="tableHeader"
+            >
+              {{ tableHeader }}
```

Comment:
Shouldn't be here `{{ $t(tableHeader) }}`?
Or data property `tableHeaders` should already contain translated texts via `i18n.t()`.

Reply:
You are right